### ISSUE TITLE
Poledit frame assignment z-then-bisector on hydrogen on aromatic ring.

### ISSUE THREAD

**Issue by walkerbdev** [TinkerTools/tinker#90]
Hello,

Noticed small issue special case with this molecule, basically benzene but replace one of C with N. 

The frame on  the hydrogen attached to N gives z and the x in the same direction and causes MD to crash.

[C5H5NH+.gdmaout.txt](https://github.com/TinkerTools/tinker/files/6736535/C5H5NH%2B.gdmaout.txt)
[C5H5NH+.key.txt](https://github.com/TinkerTools/tinker/files/6736536/C5H5NH%2B.key.txt)
[C5H5NH+-peditin.txt](https://github.com/TinkerTools/tinker/files/6736537/C5H5NH%2B-peditin.txt)

**Comment by jayponder** [comment]
Hi Brandon,

Thanks for sending this. It was indeed a small bug that would only happen for this type of pyridinium hydrogen and perhaps some other very similar things. It is now fixed in for the Tinker release/master branch on Github in a general way, and should not happen again.

Best, Jay

**Comment by bdw2292** [comment]
Thanks!

**Comment by pren** [comment]
For H(N) in aniline, the best choice is to use the bisector frame between the two C next to N.

**Comment by jayponder** [comment]
Hi Pengyu,

The case Brandon had was a pyridinum hydrogen. This is an aromatic system that is completely flat. The hydrogen on nitrogen cannot have a Z-Bisector local frame, but that's what POLEDIT gave. POLEDIT is/was aware of this situation, and the improper behavior was a bug in the planarity check for this specific situation of a monovalent atom attached to a trivalent nitrogen. It is now fixed.

As we know, trivalent nitrogens are a huge pain in AMOEBA- in my opinion they are the very worst situation for our local frame mechanisms. As we have discussed many times before, if the trivalent nitrogen can pass through planar during a calculation, then none of the local frames on either the nitrogen or the attached atoms can be Z-Bisector or Trisector.

For the H(N) in pyridinium (you say aniline above, but I think you mean pyridinium), POLEDIT will choose a Z-then-X frame to the nitrogen as the "Z" and then pick one of the two carbons as the "X". The Tinker code will not allow the kind of frame you suggest, where you use a Bisector frame involving two atoms (carbons) that are neither one bonded to the atom being defined (hydrogen). Adding that option would be kind of a mess, and I'd rather not go there. Various things depend on the local frame using a directly attached atom when one is present (the nitrogen, in the pyridinium case). Yes, if the molecule is simple symmetric pyridinium, then the current POLEDIT breaks symmetry. But it's a relatively rigid aromatic ring and the symmetry break will be tiny.
